### PR TITLE
added an animated eye moving with cursor 

### PR DESCRIPTION
**Pull request by Allennn56**

### CODE REVIEW

File: html, css, js/Animated Eye/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <div class="face">
+        <div class="eyes">
+            <div class="eye"></div>
+            <div class="eye"></div>
+        </div>
+    </div>
+    <script src="script.js"></script>
+</body>
+</html>
+Footer
```
Comment (jaswanth09SDE) [inline html, css, js/Animated Eye/index.html:20]: These Lines need to be removed.

File: html, css, js/Animated Eye/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
```
Comment (jaswanth09SDE) [inline html, css, js/Animated Eye/index.html:8]: referring style.css but there is no style.css in the folder. hence not loading the animated eye.

**Review by jaswanth09SDE — CHANGES_REQUESTED** [review]

File: html, css, js/Animated Eye/index.html (hunk 1 above)
Comment (jaswanth09SDE) [inline html, css, js/Animated Eye/index.html:20]: delete line number 20 and 21 in index.html

**Review by jaswanth09SDE — CHANGES_REQUESTED** [review]

**Comment by naveen701526** [comment]
Could you please modify the code and commit again according to @jaswanth09SDE .

**Comment by Allennn56** [comment]
Actually I am new to GitHub sir.I have tried to remove it but I am not able
to do it. I wanted to contribute 2 different projects but after opening pr
I saw that both were combined in one thing.I am not understanding what to
do ? Any help would be appreciated.
Thank you

**Comment by naveen701526** [comment]
Never mind. I have made changes to your file from my end. So will accept this pr. Thank you @Allennn56  for your contribution.

**Comment by Allennn56** [comment]
Thank you for helping me out with this 😃

**Pull request merged**